lisa/tools/fdisk.py

```diff
@@ -41,12 +44,21 @@ def make_partition(
         # fdisk -w always => always to wipe signatures
         # fdisk -W always => always to wipe signatures from new partitions
         mkfs = self.node.tools[Mkfs]
-        self.node.execute(
+        cmd_result = self.node.execute(
             f"(echo n; echo p; echo 1; echo ; echo; echo ; echo w) | "
             f"{self.command} -w always -W always {disk_name}",
             shell=True,
             sudo=True,
         )
+        if self.not_support_option_pattern.match(cmd_result.stdout):
```

Comment:
Add comments to explain what happens, and how it's fixed.

Reply:
comments added